fabric/src/main/resources/data/ad_astra/patchouli_books/astrodux/en_us/entries/the_first_launch/the_moon.json

```diff
@@ -13,7 +13,7 @@
     {
       "type": "patchouli:text",
       "title": "Returning Home",
-      "text": "To return back to earth, open your lander (shift + right-click) and collect the rocket, fuel and the launch pads. To remove the lander, punch it. Set up the launch pad, place the rocket down and fill it up with fuel. Press $(k:jump) and fly back to earth."
```

Comment:
In Ad Astra 1.0.1(or earlier I think), the launch pad was indeed a multi block structure. So it was proper to use `launch pads` with plural. But now it only takes one block of launch pad to launch, and in previous pages *astrodux* told players to only bring one launch pad for return, so it should be `launch pad` without plural.